template/src/services/axiosInstance.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+import Config from 'react-native-config'
+
+const instance = axios.create({
+  baseURL: Config.API_URL,
```

Comment:
Have you tested the config using `react-native-config`? I think we still need some other configuration before we can use this.

Reply:
yeah. This PR does not include the react-native-config. That will be configured in another ticket.

Reply:
Please remove code using it, if it hasn't been config

Reply:
@loido has created a PR for react-native-config. I feel we may keep using the environment variable cuz it hasn't effect.

Reply:
Ok clear